test/github-issues/1780/issue-1780.ts

```diff
@@ -0,0 +1,160 @@
+import "reflect-metadata";
+import {createTestingConnections, closeTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+import {Connection} from "../../../src/connection/Connection";
+import {expect} from "chai";
+import {User} from "./entity/User";
+import {MysqlDriver} from "../../../src/driver/mysql/MysqlDriver";
+import {PostgresDriver} from "../../../src/driver/postgres/PostgresDriver";
+describe("github issues > #1780 Support for insertion ignore on duplicate error", () => {
+     let connections: Connection[];
+    before(async () => connections = await createTestingConnections({
+        entities: [User],
+        schemaCreate: true,
+        dropSchema: true,
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+     let user1 = new User();
+    user1.first_name = "John";
+    user1.last_name = "Lenon";
+    user1.is_updated = "no";
+     let user2 = new User();
+    user2.first_name = "John";
+    user2.last_name = "Lenon";
+    user2.is_updated = "yes";
+     // let data = [user1, user2];
+    // Bulk insertion with duplicated data through same query with duplicate error exception is not supported in PostgreSQL
+    // https://doxygen.postgresql.org/nodeModifyTable_8c_source.html : Line 1356
+     it("should save one row without duplicate error in MySQL/MariaDB", () => Promise.all(connections.map(async connection => {
+      try {
+         if (connection.driver instanceof MysqlDriver) {
```

Comment:
We're creating connection to every db engine supported by typeorm but using only three. Reloading is also not necessary since we use different driver in different test.
It isn't much difference but it will take another second or two if every supported db engine will be tested.